nautobot/extras/urls.py

```diff
@@ -410,6 +410,11 @@
         views.RelationshipBulkDeleteView.as_view(),
         name="relationship_bulk_delete",
     ),
+    path(
+        "relationships/<uuid:pk>/",
```

Comment:
Since Relationships *have* a slug, we really should be using that rather than the PK here, but I see you're following an established pattern with the other Relationship URL patterns.

Reply:
Since the relationship model doesn't appear to be a full first class citizen from a standards perspective AND the UI views are limited I can swap out the patterns for all the views if you would like.

Reply:
All done